css-brochure-form/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Brochure Form</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+
+</head>
+<body>
+  <h1> LEARN MORE </h1>
+  <form action="http://www.example.com/signup.php">
+    <p>COMPANY EMAIL</p>
```

Comment:
each of these should be labels, and make sure there is a matching for and id attribute to maintain label functionality.